scripts.js

```diff
@@ -105,4 +90,52 @@ const resetBoard = () => {
     // sets the innerHTML to null to replace the "X" or "O"
     squares[i].innerHTML = null
   }  
+
+}
+
+const checkForWin = () => {
+  if(horizontalWin() || verticalWin() || diagonalWin()) {
+    window.alert(`Player ${currentMarker} won!`)
+  } else { console.log('else')
+    changeMarker()
+  }
+}
+
+const horizontalWin = () => {
+  // Your code here to check for horizontal wins
+    if((board[0][0] == "X" && board[0][1] == "X" && board[0][2] == "X")
+    || (board[0][0] == "O" && board[0][1] == "O" && board[0][2] == "O")
+    ){return true}
```

Comment:
lets put the return on a new line and then the closing } on it's own line too.
`if() {`
` return true`
`}`

Reply:
putting it on it's own line as 
if() {
return true
}
resulted in it not working for some reason but doing it without the if() worked